### PR TITLE
Simplify env vars

### PR DESCRIPTION
**Pull request by samajammin**
Small change but @CarlBeek would like to get your review to ensure this won't screw anything up e.g. on the Netlify side where we may set ENV variables.

Just trying to consolidate the places where we specify the testnet name & deduce from `IS_MAINNET` when possible so it's easier to update when the time comes.

### CODE REVIEW

**Comment by samajammin** [comment]
Hey @CarlBeek checking in on this. Thanks!

File: src/utils/envVars.ts [hunk 1]
```diff
@@ -7,7 +7,10 @@ export const ENABLE_RPC_FEATURES        = Boolean(INFURA_PROJECT_ID && INFURA_PR
 export const INFURA_URL                 = `https://${IS_MAINNET ? "mainnet" : "goerli"}.infura.io/v3/${INFURA_PROJECT_ID}`;
 
 // public
-export const ETH2_NETWORK_NAME          = process.env.REACT_APP_ETH2_NETWORK_NAME   || 'Mainnet';
+export const TESTNET_LAUNCHPAD_NAME      = 'Pyrmont'
+export const TESTNET_LAUNCHPAD_URL      = `https://${TESTNET_LAUNCHPAD_NAME.toLowerCase()}.launchpad.ethereum.org/`
+export const MAINNET_LAUNCHPAD_URL      = 'https://launchpad.ethereum.org/'
+export const ETH2_NETWORK_NAME          = IS_MAINNET ? TESTNET_LAUNCHPAD_NAME : 'Mainnet';
```
Comment (CarlBeek) [inline src/utils/envVars.ts:13]: I think this is backwards, shouldn't it be:
```suggestion
export const ETH2_NETWORK_NAME          = IS_MAINNET ? 'Mainnet': TESTNET_LAUNCHPAD_NAME ;
```

File: src/utils/envVars.ts [hunk 2]
```diff
@@ -7,7 +7,10 @@ export const ENABLE_RPC_FEATURES        = Boolean(INFURA_PROJECT_ID && INFURA_PR
 export const INFURA_URL                 = `https://${IS_MAINNET ? "mainnet" : "goerli"}.infura.io/v3/${INFURA_PROJECT_ID}`;
 
 // public
-export const ETH2_NETWORK_NAME          = process.env.REACT_APP_ETH2_NETWORK_NAME   || 'Mainnet';
+export const TESTNET_LAUNCHPAD_NAME      = 'Pyrmont'
```
Comment (CarlBeek) [inline src/utils/envVars.ts:10]: I think this should still be configurable via a `process.env.` env-var as there are times when we have multiple concurrent testnets and ideally, the launchpad is not where this is configured. You shouldn't have to change the source code to change the testnet name.

**Review by CarlBeek — COMMENTED** [review]
I do think there is some cleaning up to do here, but I am not convinced that this is the way to do it. IMO, you shouldn't have to go into the source code to configure a testnet name. (Especially as sometimes there are multiple concurrent testnets).

**Pull request closed without merging**